packages/warriorjs-core/src/Space.test.js

```diff
@@ -123,6 +123,10 @@ describe('Space', () => {
       expect(space.isEnemy()).toBe(false);
```

Comment:
We should remove this test.

Reply:
Why is that? Should I remove `.isEnemy` like I just did with `.isCaptive` instead of deprecating it  and remove all of his tests

Reply:
Yes, that is correct.